### PR TITLE
Partial for error messages

### PR DESCRIPTION
**Pull request by DimaSmagulov**
code review:
- [ ] @CHILL99 
- [ ] @krisberry 
- [ ] @natalya5 
- [x] @oleg08 
- [ ] @OrestF 
- [x] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [ ] @Kmalynss
- [ ] @serhiym 
- [x] @ruba-ruba

1. Added partial for error messages

### CODE REVIEW

**Comment by vvs79** [comment]
reviewed

File: app/views/shared/_error_messages.html.slim [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- if object.errors.any?
+      .alert.alert-danger
+        a.close data-dismiss="alert" href="#"  ×
+        h4
+          = pluralize(object.errors.count, 'error')
+        ul
+          - object.errors.full_messages.each do |message|
+            li
+              = message
```
Comment (OrestF) [inline app/views/shared/_error_messages.html.slim]: missing 'new line' at the end

Comment (DimaSmagulov) [inline app/views/shared/_error_messages.html.slim]: cop said nothing)

**Comment by rtershak** [comment]
reviewed

**Comment by oleg08** [comment]
reviewed

File: app/views/shared/_error_messages.html.slim (hunk 1 above)
Comment (Kmalynss) [inline app/views/shared/_error_messages.html.slim]: let's leave this to rubocop

**Pull request merged**